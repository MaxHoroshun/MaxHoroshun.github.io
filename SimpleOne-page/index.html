<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Site HTML5/CSS</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>

	<header>

		<div id="logo" onclick="slowScroll('#top')">
			<span>ITMaster</span>
		</div>

		<div id="about">
			<a href="#" title="Возможности" onclick="slowScroll('#main')">Возможности</a>
			<a href="#" title="Преимущества" onclick="slowScroll('#overview')">Преимущества</a>
			<a href="#" title="Контакты" onclick="slowScroll('#contacts')">Контакты</a>
			<a href="#" title="faq" onclick="slowScroll('#faq')">FAQ</a>
		</div>

	</header>

	<div id="top">
		<h1>Программирование</h1>
		<h3>как способ жизни!</h3>
	</div>

	<div id="main">
		<div class="intro">
			<h2>Наши услуги помогут вам!</h2>
			<span>Большой выбор всего, что может вам пригодиться</span>
		</div>
		<div class="text">
			<span>Выбор товара по всем наименованиям и брендам со всего мира! У нас Вы найдете все! Большой ассортимент товаров из Европы и США!</span>
		</div>
	</div>

	<div id="overview">
		<h2>Преимущества</h2>
		<h4>С нами будет легче!</h4>

		<div class="img">
			<img src="https://i.pinimg.com/564x/61/d6/91/61d691a82a4d85eec6b9c55d068b6050.jpg" alt="Ruby">
			<span>Ищучение языка Ruby для начинающих.</span>
		</div>

		<div class="img">
			<img src="http://cdn2.iconfinder.com/data/icons/flurry-for-creative-suite/512/photoshop2.png" alt="Photoshop">
			<span>Уроки Photoshop для начинающих.</span>
		</div>

		<div class="img">
			<img src="https://i.pinimg.com/564x/d5/bd/0a/d5bd0a9d074aad02e42c87ac9567d700.jpg" alt="Swift">
			<span>Уроки Swift для начинающих.</span>
		</div>

		<div class="img">
			<img src="https://itea.ua/wp-content/uploads/2015/04/python-2-min.png" alt="Python3">
			<span>Ищучение языка Python 3 для начинающих.</span>
		</div>
	</div>

	<div id="contacts">
		<center><h5>Обратная связь</h5></center>
		<form id="form_input">
			<label for="name">Имя <span>*</span></label><br>
			<input type="text" placeholder="Введите имя" name="name" id="name"><br>
			<label for="email">Ваша почта <span>*</span></label><br>
			<input type="email" placeholder="Введите вашу почту" name="email" id="email"><br>
			<label for="message">Сообщение <span>*</span></label><br>
			<textarea placeholder="Введите ваше сообщение" name="message" id="message"></textarea><br>
			<div id="mess_send" class="btn">Отправить</div>
		</form>
	</div>

	<div id="faq">
		<div>
			<span class="title">Оплата</span><br>
			<span class="heading">Как будет проходить оплата?</span>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui quae excepturi, voluptatem atque rem possimus dolor ducimus commodi exercitationem quaerat voluptate quam iste maxime corporis illum quos, molestiae eaque nihil.</p>
		</div>
		<div>
			<span class="title">Информация</span><br>
			<span class="heading">Что входит в услуги сервиса?</span>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui quae excepturi, voluptatem atque rem possimus dolor ducimus commodi exercitationem quaerat voluptate quam iste maxime corporis illum quos, molestiae eaque nihil.</p>
		</div>
		<div>
			<span class="title">Гарантии</span><br>
			<span class="heading">Какие гарантии есть?</span>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui quae excepturi, voluptatem atque rem possimus dolor ducimus commodi exercitationem quaerat voluptate quam iste maxime corporis illum quos, molestiae eaque nihil.</p>
		</div>
	</div>

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
	<script>
		function slowScroll(id) {
			$('html, body').animate({
				scrollTop: $(id).offset().top
			}, 500);
		}

		$(document).on("scroll", function () {
			if($(window).scrollTop() === 0)
				$("header").removeClass("fixed");
			else
				$("header").attr("class", "fixed");
		});
	</script>
	
</body>
</html>