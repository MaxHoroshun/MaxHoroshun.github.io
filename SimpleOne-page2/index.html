<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<title>I want to Code</title>
	<link href="https://fonts.googleapis.com/css?family=Montserrat:200,400,500,900&display=swap&subset=cyrillic" rel="stylesheet">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	
	<header class="header">
		<div class="container">
			<nav class="nav">
				<img src="img/logo.png" alt="logo" class="logo">
				<ul class="menu">
					<li><a href="#">Главная</a></li>
					<li><a href="#">Новости</a></li>
					<li><a href="#">Каталог</a></li>
					<li><a href="#">Гарантии</a></li>
					<li><a href="#">О Нас</a></li>
				</ul>
				<div class="phone">
					+38 050 450 50 75
				</div>
			</nav>
			<div class="offer">
				<div class="offer-content">
					<h1>Я хочу<br>научиться верстать!</h1>
					<p>HTML5 CSS JS</p>
					<a href="#">Свяжитесь со мной!</a>
				</div>
				<div class="triangles">
					
				</div>
			</div>
		</div>
	</header>

	<script src="js/jquery-3.4.1.min.js"></script>
	<script src="js/main.js"></script>

</body>
</html>