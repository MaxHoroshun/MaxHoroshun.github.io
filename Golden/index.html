<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Golden</title>
  <link rel="stylesheet" href="css/style.css">
</head>
<body>
  <header>
    <div class="container">
      <div class="heading clearfix">
        <img src="img/logo.png" alt="Golden" class="logo">
    
            <nav>
                <ul class="menu">
                  <li><a href="#">Home</a></li>
                  <li><a href="#">Services</a></li>
                  <li><a href="#">Portfolio</a></li>
                  <li><a href="#">About</a></li>
                  <li><a href="#">Contact</a></li>

                </ul>
            </nav>
      </div>
        <div class="titles">
          <div class="titles__first">
            Welcome to Our Studio!
          </div>
            <h1>It's nice to meet you</h1>
        </div>
          <a class="button" href="#">Tell me more</a>

    </div>
  </header>
    
  <section id="services">
    <div class="container">
        <div class="title">
          <h2>
            Services
          </h2>
          <p>Какая-то интересная строка</p>
        </div>
      <div class="services clearfix">
        <div class="services__item">
          <img src="img/icon1.png" alt="Услуга">
          <h3>E-Commerce</h3>
          <p>Много текста Много текста Много текста Много текста Много текста
          Много текста Много текста</p>
        </div>
        <div class="services__item">
          <img src="img/icon2.png" alt="Услуга">
          <h3>Responsive Web</h3>
          <p>Много текста Много текста Много текста Много текста Много текста
          Много текста Много текста</p>
        </div>
        <div class="services__item">
          <img src="img/icon3.png" alt="Услуга">
          <h3>Web Security</h3>
          <p>Много текста Много текста Много текста Много текста Много текста
          Много текста Много текста</p>
        </div>
      </div>
    </div>
  </section>

  <section id="Portfolio">
    <div class="container">
      <div class="title">
          <h2>
            Our Portfolio
          </h2>
          <p>Какая-то интересная строка</p>
        </div>
        <div class="works clearfix">
          <img src="img/img1.jpg" alt="Работа">
          <img src="img/img2.jpg" alt="Работа">
          <img src="img/img3.jpg" alt="Работа">
        </div>
    </div>
  </section>

  <section>
    <div class="container">
      <div class="about_us">
        <div class="about">
          <h1>About us</h1>
          <p>Какой-то интересный текст</p>
        </div>
    <div class="achieves">
      <div class="achieve__marg">
      <div class="achieve__item">
        <h3>July 2010 <br>
          Our Humble Beginnings</h3>
        <p>Proin iaculis purus consequat sem cure 
            digni ssim. Donec porttitora entum suscipit 
            aenean rhoncus posuere odio in tincidunt. Proin 
            iaculis purus consequat sem cure digni 
            ssim. Donec porttitora entum suscipit.</p>     
      </div>
      <div class="achive__img"><img src="img/about1.jpg" alt="Июль"></div>
      </div>
      
      <div class="achieve__marg">
        <div class="achieve__item2">
          <h3>January 2011 <br>
            Facing Startup Battles</h3>
          <p>Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit.</p>     
        </div>
        <div class="achive__img"><img src="img/about2.jpg" alt="Январь"></div>
      </div>
      <div class="achieve__marg">
        <div class="achieve__item">
          <h3>December 2012 <br>
            Enter The Dark Days</h3>
            <p>Proin iaculis purus consequat sem cure 
            digni ssim. Donec porttitora entum suscipit 
            aenean rhoncus posuere odio in tincidunt. Proin 
            iaculis purus consequat sem cure digni 
            ssim. Donec porttitora entum suscipit.</p>
        </div>
        <div class="achive__img"><img src="img/about3.jpg" alt="Декабрь"></div>
      </div>
    <div class="achieve__marg">
      <div class="achieve__item2">
        <h3>february 2014
          Our Triumph<br>
            Facing Startup Battles</h3>
            <p>Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit.</p>
      </div>
      <div class="achive__img"><img src="img/about4.jpg" alt="Февраль"></div>
    </div>
      </div>
    </div>
      </div>
    </div>
  </section>


  <footer>
      <div class="container">
        <p id="copyright">&copy; Copyright 2017 FreebiesXpress.com</p>
      </div>
  </footer>

        

</body>
</html>